### ISSUE TITLE
Retorno ao executar

### ISSUE THREAD

**Issue by rodrigobragamg** [Nucito/CaptchaSolver#1]
Obtive o seguinte erro (python 3.9.9, todos módulos instalados, executando como admin):

c:\CaptchaSolver-main>python captcha_solver.py

                 from  n    params  module                                  arguments
  0                -1  1      3520  models.common.Focus                     [3, 32, 3]
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]
  2                -1  1     19904  models.common.BottleneckCSP             [64, 64, 1]
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]
  4                -1  1    161152  models.common.BottleneckCSP             [128, 128, 3]
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]
  6                -1  1    641792  models.common.BottleneckCSP             [256, 256, 3]
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]
  8                -1  1    656896  models.common.SPP                       [512, 512, [5, 9, 13]]
  9                -1  1   1248768  models.common.BottleneckCSP             [512, 512, 1, False]
 10                -1  1    131584  models.common.Conv                      [512, 256, 1, 1]
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 12           [-1, 6]  1         0  models.common.Concat                    [1]
 13                -1  1    378624  models.common.BottleneckCSP             [512, 256, 1, False]
 14                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 16           [-1, 4]  1         0  models.common.Concat                    [1]
 17                -1  1     95104  models.common.BottleneckCSP             [256, 128, 1, False]
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]
 19          [-1, 14]  1         0  models.common.Concat                    [1]
 20                -1  1    313088  models.common.BottleneckCSP             [256, 256, 1, False]
 21                -1  1    590336  models.common.Conv                      [256, 256, 3, 2]
 22          [-1, 10]  1         0  models.common.Concat                    [1]
 23                -1  1   1248768  models.common.BottleneckCSP             [512, 512, 1, False]
 24      [17, 20, 23]  1     40455  models.yolo.Detect                      [10, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 256, 512]]
C:\Users\Rodrigo\AppData\Local\Programs\Python\Python39\lib\site-packages\torch\functional.py:445: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  ..\aten\src\ATen\native\TensorShape.cpp:2157.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Model Summary: 283 layers, 7279367 parameters, 7279367 gradients, 16.9 GFLOPS

Adding autoShape...
{'Captcha': '861', 'Cods': [{'6': {'x1': 47.03, 'y1': 278.7, 'x2': 159.94, 'y2': 357.61, 'per': 0.91}}, {'1': {'x1': 37.01, 'y1': 366.24, 'x2': 180.87, 'y2': 399.01, 'per': 0.91}}, {'8': {'x1': 47.16, 'y1': 195.8, 'x2': 152.64, 'y2': 263.24, 'per': 0.85}}]}

**Comment by Nucito** [comment]
@rodrigobragamg Isso não seria um erro o Pytorch esta dando esse alerta por conta do log dele, vou enviar uma atualização para fazer o suppress desse log 😁

**Comment by rodrigobragamg** [comment]
> @rodrigobragamg Isso não seria um erro o Pytorch esta dando esse alerta por conta do log dele, vou enviar uma atualização para fazer o suppress desse log 😁

Verdade Nucito, usei o termo errado.

Mas do jeito que você programou, ele resolve 1 captcha e encerra ou fica aguardando e resolvendo em lopp?

**Comment by fguide** [comment]
Veja a página principal, está descrito que não resolve mas sim identifica, é um código para você adaptar ao seu uso.

Se não souber como fazer, sugiro que aguarde o bot oficial que sairá com o código integrado.

**Comment by Nucito** [comment]
@rodrigobragamg Ele resolve apenas 1 captcha e para, eu apenas coloquei um exemplo no código para poder deixar como uma prova de projeto kkkkkk
